jaeger-core/src/main/java/io/jaegertracing/internal/propagation/TraceContextCodec.java

```diff
@@ -34,8 +34,8 @@
 @Slf4j
 public class TraceContextCodec implements Codec<TextMap> {
 
-  static final String TRACE_PARENT = "traceparent";
-  static final String TRACE_STATE = "tracestate";
+  public static final String TRACE_PARENT = "traceparent";
```

Comment:
These names are defined in the W3C specification. You can simply define them in your codebase.